docs/learn/about-iota/why-is-iota-feeless.md

```diff
@@ -5,18 +5,45 @@ description: The unique feature of IOTA is the absence of any Fee's in the proto
+
+This race to mine the next block is a key security feature in blockchain. If one could mine blocks at will, an attacker could make a competing blockchain and, by getting enough nodes to publish it, could create fraudulent transactions, reverse legitimate ones, or any of a number of other things that would rapidly cause trust in the system and the value of the associated coin to trend to 0.
+
+Because each block in the blockchain contains a double hash of the header of the previous block, any change made earlier in the blockchain requires all subsequent blocks be changed also, and these changes require the newly-changed blocks be mined again. It is easy to see that this system provides security not only for new transactions, but that the older a transaction in the blockchain is, the more secure it becomes.
+
+So miners are critical to blockchain, but mining is expensive work and must be financially compensated.
+
+When a block is mined, the successful miner is allowed to add a number of coins -- the block reward -- to his or her own wallet, the number being set by the protocol. These newly minted coins can be sold by the miner and are currently the primary way the miners are paid for their work. Unfortunately for miners, many crypto protocols reduce the size of the block reward over time. But there is another way miners can still get paid.  This is where the user comes in.
+
+The role of the user is simple. They buy, sell, and trade the currency native to their chosen blockchain. If blockchain transactions were feeless, the user needs only use their private key to sign a transaction putting a set value of crypto under the control of the recipient whose public key is included in the transaction.
+
+But there is a problem for the user here. The target difficulty of mining blocks is designed to force new blocks to be mined at a relatively constant rate. And the blocks themselves usually have a fixed size which in turn fixes a limit on the number of transactions that can be included per block. If there are more transactions waiting to be included in a block, the miners will usually select transactions that include a payment, a tip, to whomever mines the block the transaction is included in. The higher the fee, the sooner selected.
+
+No solution has been proposed yet on blockchain that can get miners to adequately secure the system but which doesn't make users pay for it in some way.
+
+
+## IOTA and the DAG
+
+To do. Users and miners are one and the same on a DAG (setting aside outsourced POW) and the POW itself is many orders of magnitude less difficult. Random tip selection ensures a fair and equal chance for any transaction to be the next one added to the DAG . . .
```

Comment:
To do. Users and miners are the same in a DAG (setting aside outsourced POW) and the POW itself is many orders of magnitude less difficult. Random tip selection ensures a fair and equal chance for any transaction to be the next one added to the DAG . . .